aspnetcore/security/authentication/azure-ad-b2c.md

```diff
@@ -99,6 +99,13 @@ Use the steps in the Azure AD B2C documentation to [create a sign-up or sign-in
 > [!WARNING]
 > Ensure the policy names are exactly as described in the documentation, as those policies were used in the **Change Authentication** dialog in Visual Studio. The policy names can be verified in *appsettings.json*.
 
+## Configuring the underlying OpenIdConnectOptions options
+
+If you need to configure the OpenIdConnectOptions directly, you can configure them via:
+
+`serviceCollection.Configure<OpenIdConnectOptions>(AzureAD[B2C]Defaults.OpenIdScheme, o => { ... }`
```

Comment:
There's also a JwtBearer version of this, right?

Reply:
Yeah i can tweak this to be more general, since it applies for cookies/jwtbearer as well